FAQs/README.md

```diff
@@ -160,9 +161,17 @@ In this case, we recommend you to upgrade the Function app hosting plan (pricing
 4. To increase function timeout, go to your function app (say, you want to increase timeout value for 'AzSK-AzTS-WorkItemProcessor-xxxxx'. This app contains function to scan subscription with baseline control.) > Settings > Configuration > Application settings > Update the value of `AzureFunctionsJobHost__functionTimeout` to '01:00:00' to increase the timeout value to 1 hour.
 
   > _**Note:** In future if you run the AzTS installation command (`Install-AzSKTenantSecuritySolution`) to upgrade your existing AzTS setup, you will have to repeat the above steps._
-<br> 
-<br>
 
+### **4. How do I pull data from AzTS scan and push it to different data sources?**
+
+Azure Tenant Security Solution centrally scans subscriptions and gets control scan results, processed RBAC data, resource inventory data, subscription metadata, etc., and pushes all the data into a central storage account.<br>
```

Comment:
into a "central storage account" (How to identify this storage account?)

Reply:
Added name of storage account.